cookiecutter/vcs.py

```diff
@@ -81,7 +81,7 @@ def clone(repo_url, checkout=None, clone_to_dir='.', no_input=False):
     if repo_type == 'git':
         repo_name = repo_name.split(':')[-1].rsplit('.git')[0]
         repo_dir = os.path.normpath(os.path.join(clone_to_dir, repo_name))
-    elif repo_type == 'hg':
+    if repo_type == 'hg':
```

Comment:
Because the implicit `else` create a new untestable branch.

```
if C not in ["A", "B"]:
   return

if C == "A":
    ...
elif == "B":
    ...
```

Coverage expects a test for the empty else case (so not "A" and not "B"), but you can't write the test case because of `C not in ["A", "B"]`.

Reply:
lol

Good to know, thanks!